packages/react-router-dom-v5-compat/lib/components.tsx

```diff
@@ -14,10 +14,10 @@ import { Router, Routes, Route } from "../react-router-dom";
 // v5 isn't in TypeScript, they'll also lose the @types/react-router with this
 // but not worried about that for now.
 export function CompatRoute(props: any) {
-  let { path } = props;
+  let { location, path } = props;
   if (!props.exact) path += "/*";
   return (
-    <Routes>
+    <Routes {...(location && { location })}>
```

Comment:
Any reason this can't just be `<Routes location={location}>`?

Reply:
I just wanted to play it safe, but I've just seen that `Routes` checks for truthiness on `location` before using it:

https://github.com/remix-run/react-router/blob/715dd233bb57a65c563edd52c4ccd63f37745ddb/packages/react-router/lib/hooks.tsx#L322-L324

So I'll go with your suggestion 🙃